src/containerapp/azext_containerapp/_params.py

```diff
@@ -66,6 +66,10 @@ def load_arguments(self, _):
         c.argument('managed_certificates_only', options_list=['--managed-certificates-only', '-m'], help='List managed certificates only.')
         c.argument('private_key_certificates_only', options_list=['--private-key-certificates-only', '-p'], help='List private-key certificates only.')
 
+    with self.argument_context('containerapp env dapr-component init') as c:
+        c.argument('statestore', help="The state store component and dev service to create.")
```

Comment:
could you use argument --state-store which will be easy to view for customer like what we do for most arguments? For arguments with multiple words should be separated with `-`

Reply:
Hi, for users coming from the Dapr world, they will be more familiar with `statestore` and `pubsub`. Also please check https://github.com/Azure/azure-cli-extensions/pull/6857#discussion_r1375754681